site/en/blog/100-web-moments/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Celebrate Chrome 100 with #100CoolWebMoments'
+description: >
+  We are celebrating the launch of Chrome version 100 with a walk down memory lane. 
+layout:
+  'layouts/blog-post.njk'
+date: 2022-03-29
+authors:
+  - rachelandrew
+hero: 'image/kheDArv5csY6rvQUJDbWRscckLr1/AlyVvShYHBM5I7ikG8vj.jpg'
+alt: >
+  100 cool web moments.
+tags:
+  - chrome-100
+---
+
+Google Chrome launched in 2018, and in March 2022 we are celebrating the launch of Chrome version 100 with a walk down memory lane. [#100CoolWebMoments](/100/) remembers some cool, fun, interesting, and important things that have happened in the past 14 years.
```

Comment:
FWIW, @harleenkbatra08 was happy with the "scroll down memory lane" language usage elsewhere, instead of "walk down memory lane". (Because you _scroll_ down the page?) Total nit, though.